stage1/rootfs/usr_from_host/Makefile

```diff
@@ -1,13 +1,19 @@
 # do not include systemd in stage1, let's find it at run-time
 
-usr.done install: Makefile
+usr.done: Makefile install
+	cp -f install ../aggregate/install.d/00usr
+	touch usr.done
```

Comment:
why are you using this flag file instead of just referencing the target "../aggregate/install.d/00usr"?